### PR TITLE
Health Education Page

### PR DESCRIPTION
**Pull request by solemabrothers**

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml [hunk 1]
```diff
@@ -0,0 +1,188 @@
+<htmlform formName="HMIS 003 HIV Care ART Card - Health Education And Psychosocial Form"
+          formDescription="Captures health eduction and counseling encounters"
+          formEncounterType="6d88e370-f2ba-476b-bf1b-d8eaf3b1b67e"
+          formUuid="b8224534-7df0-4c00-ba8e-6b1f291087eb"
+          formVersion="1.2"
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml]: @solemabrothers change version to 3.0

File: omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+                    <div class="col-">
+                        Is Visit On Schedule <obs id="br" conceptId="90069" style="checkbox"/>
+
+                    </div>
+                </div>
+
+            </div>
+
+        </div>
+        <div class="card-body">
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml]: @solemabrothers remove empty card-body div

File: omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+          formIcon="icon-medkit"
+          formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false &amp;&amp; sessionLocation.uuid=='86863db4-6101-4ecf-9a86-5e716d6504e4'"
+          formDisplayStyle="Standard"
+          formLabel="ART Card -  Health Education And Psychosocial Support Form">
+
+
+
+
+    <div class="card">
+        <div class="headers" style="text-align: center;">
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml]: @solemabrothers use class=card-header, and add bootstrap center align class

File: omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+          formUuid="b8224534-7df0-4c00-ba8e-6b1f291087eb"
+          formVersion="1.2"
+          formAddMetadata="yes"
+          formUILocation="patientDashboard.visitActions"
+          formOrder="2"
+          formIcon="icon-medkit"
+          formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false &amp;&amp; sessionLocation.uuid=='86863db4-6101-4ecf-9a86-5e716d6504e4'"
+          formDisplayStyle="Standard"
+          formLabel="ART Card -  Health Education And Psychosocial Support Form">
+
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml:14]: @solemabrothers remove empty spaces

File: omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+            </div>
+            <div class="card-body">
+                <div class="row">
+                    <div class="col-6">
+                        <div class="row">
+                        Visit Date  <encounterDate default="today"  disallowMultipleEncountersOnDate="block" />
+                        </div>
+                    </div>
+                    <div class="col-">
+                        Is Visit On Schedule <obs id="br" conceptId="90069" style="checkbox"/>
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml]: @solemabrothers remove id="br" which should be unique

File: omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+            <div class="card-header">
+                Encounter Information
+            </div>
+
+            <div class="card-body">
+                <table class="table table-striped">
+                    <tr>
+                        <td>Settings</td>
+                        <td>
+                            <div class="row">
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml]: @solemabrothers Remove div class="row" in all table rows

Comment (solemabrothers) [reply omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml]: @ssmusoke  This has been done, however, i had to include an internal styling code for the alignment of radio buttons

**Comment by ssmusoke** [comment]
@solemabrothers 

1. Setting - add the codes before each of the values as per the guidelines
2. PSS Issues identified - add the codes before each of the values as per the guidelines 
3. ART Preparation - missing before 
4. Depression status - use the full text on the card 

See final version of this card on Asana and correct typos

**Pull request merged**